test/setup.ts

```diff
@@ -34,13 +34,13 @@ const setupTest = deployments.createFixture<
   const { deployer, user, anon } = await getNamedAccounts();
 
   // Contracts
-  const minter: HypercertMinterV0 = await ethers.getContract("HypercertMinterV0");
+  const minter: HypercertMinterV1 = await ethers.getContract(HypercertMinter_V1);
```

Comment:
Same thing - define a type that points to the current version and then there is only one line to update each time?

Reply:
done